packages/at_auth/lib/src/auth/models/at_auth_responses.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:at_auth/src/keys/at_keys.dart';
+
+sealed class AuthResponse {
```

Comment:
This needs to return AtClient on success, otherwise we cannot get rid of singleton later.